src/tests/system/tests/test_local_overrides.py

```diff
@@ -0,0 +1,263 @@
+        3. Authenticate as "user1", short and fully qualified name
+        4. Authenticate as "o-user1", short and fully qualified name
+        5. Authenticate as "lo-user1", short and fully qualified name
+        6. Search for user local overrides
+    :expectedresults:
+        1. User local override is created
+        2. Nested user local override is created
+        3. Authentication successful for both short and fully qualified name
+        4. Authentication successful for both short and fully qualified name
+        5. Authentication successful for both short and fully qualified name
+        6. Local overrides is found for all users
+    :customerscenario: False
+    """
+    provider.user("user1").add(
+        uid=999011, gid=999011, home="/home/user1", gecos="user", shell="/bin/bash", password="Secret123")
+    client.sssd.start()
+    assert client.ssh("user1", "Secret123")
+    client.local_override.user("user1").add(name="o-user1")
+    client.sssd.restart()
+    client.local_override.user("user1").add(name="lo-user1")
+    assert client.ssh("user1", "Secret123")
+    assert client.ssh("o-user1", "Secret123")
+    assert client.ssh("lo-user1", "Secret123")
+    assert client.ssh(f"user1@{client.sssd.domain}", "Secret123")
+    assert client.ssh(f"o-user1@{client.sssd.domain}", "Secret123")
+    assert client.ssh(f"lo-user1@{client.sssd.domain}", "Secret123")
+
+    client.local_override.user("user1").delete()
+    assert client.local_override.user("user1").get() == {}
+
+
+@pytest.mark.topology(KnownTopology.LDAP)
+@pytest.mark.topology(KnownTopology.AD)
+def test_local_overrides__import_export(client: Client, provider: GenericProvider):
+    """
+    :title: Export and import local overrides
+    :setup:
+        1. Create posix user "user1" with posix attributes defined
+        2. Start SSSD
+    :steps:
+        1. Override user "user1" to "o-user1" and group "group1" to "o-group1"
+        2. Search for overrides
+        3. Export user and group local overrides data to a file
+        4. Delete overrides
+        5. Import user and group local overrides data
+        6. Search for user and group local overrides
+    :expectedresults:
+        1. User local override is created
+        2. User local overrides is found
+        3. Local overrides data is exported to a file
+        4. Local override is deleted
+        5. Local override data is imported from file
+        6. Local overrides are found
+    :customerscenario: False
+    """
+
+    provider.user("user1").add(
+        uid=999011, gid=999011, home="/home/user1", gecos="user", shell="/bin/bash", password="Secret123")
+    provider.group("group1").add(gid=999999)
+    client.sssd.start()
+    assert client.ssh("user1", "Secret123")
+    client.local_override.user("user1").add(name="o-user1")
+    client.local_override.group("group1").add(name="o-group1")
+    client.sssd.restart()
+
+    client.local_override.user("user1").backup()
```

Comment:
if backup() does an actual command export, it may be beneficial to someone reading the code to add a comment above this line stating something like:

`# local_userride.user().backup() performs the override export for all users`

Also, just to clarify, backup() from [here](https://github.com/SSSD/sssd-test-framework/pull/8) appears to export all users (likewise for groups later).  Correct?

Reply:
Done.